python_modules/dagster-graphql/dagster_graphql/implementation/execution/backfill.py

```diff
@@ -182,6 +198,26 @@ def create_and_launch_partition_backfill(
             ),
             all_partitions=backfill_params.get("allPartitions", False),
         )
+    elif partitions_by_assets is not None:
+        asset_graph = ExternalAssetGraph.from_workspace(graphene_info.context)
+        _assert_permission_for_asset_graph(
+            graphene_info, asset_graph, asset_selection, Permissions.LAUNCH_PARTITION_BACKFILL
+        )
```

Comment:
just like for the asset selection backfills above, I think we want to raise an exception if `forceSynchronousSubmission` and `fromFailure` are provided

Reply:
thanks! added those check now